modules/core/src/adapters/deck-adapter.js

```diff
@@ -148,4 +168,8 @@ export default class DeckAdapter {
       proceedToNextFrame(nextTimeMs);
     });
   }
+
+  _onEnabled() {
```

Comment:
Since this function is one line and we only enable once or twice, could you remove the function and do direct assignments instead?